bindings/js/api.json

```diff
@@ -1518,6 +1517,407 @@
+          ]
+        },
+        {
+          "name": "msgHash",
+          "kind": "input",
+          "type": "buffer",
+          "size": 32,
+          "interface_index": 0,
+          "tests": [
+              "4b688df40bcedbe641ddb16ff0a1842d9c67ea1c3bf63f3e0471baa664531d1a"
+          ]
+        },
+        {
+          "name": "private_key",
+          "kind": "input",
+          "type": "buffer",
+          "size": 32,
+          "interface_index": 1,
+          "tests": [
+              "ebb2c082fd7727890a28ac82f6bdf97bad8de9f5d7c9028692de1a255cad3e0f"
+          ]
+        },
+        {
+          "name": "nonce",
+          "kind": "input",
+          "type": "buffer",
+          "size": 32,
+          "interface_index": 2,
+          "tests": [
+              "49a0d7b786ec9cde0d0721d72804befd06571c974b191efb42ecf322ba9ddd9a"
+          ]
+        }
+      ],
+      "return": {
+        "type": "bool",
+        "tests": [
+            "true"
+        ]
+      }
+    },
+    "ecdsa_verify_hashed_msg": {
+      "module": "Hacl_K256_ECDSA",
+      "name": "ecdsa_verify_hashed_msg",
+      "args": [{
+          "name": "m",
+          "kind": "input",
+          "type": "buffer",
+          "size": 32,
+          "interface_index": 0,
+          "tests": [
+              "4b688df40bcedbe641ddb16ff0a1842d9c67ea1c3bf63f3e0471baa664531d1a",
+              "4b688df40bcedbe641ddb16ff0a1842d9c67ea1c3bf63f3e0471baa664531d1a"
+          ]
+        },
+        {
+          "name": "public_key",
+          "kind": "input",
+          "type": "buffer",
+          "size": 64,
+          "interface_index": 1,
+          "tests": [
+              "779dd197a5df977ed2cf6cb31d82d43328b790dc6b3b7d4437a427bd5847dfcde94b724a555b6d017bb7607c3e3281daf5b1699d6ef4124975c9237b917d426f","779dd197a5df977ed2cf6cb31d82d43328b790dc6b3b7d4437a427bd5847dfcde94b724a555b6d017bb7607c3e3281daf5b1699d6ef4124975c9237b917d426f",
+              "779dd197a5df977ed2cf6cb31d82d43328b790dc6b3b7d4437a427bd5847dfcde94b724a555b6d017bb7607c3e3281daf5b1699d6ef4124975c9237b917d426f","779dd197a5df977ed2cf6cb31d82d43328b790dc6b3b7d4437a427bd5847dfcde94b724a555b6d017bb7607c3e3281daf5b1699d6ef4124975c9237b917d426f"
+          ]
+        },
+        {
+          "name": "signature",
+          "kind": "input",
+          "type": "buffer",
+          "size": 64,
+          "interface_index": 2,
+          "tests": [
+              "241097efbf8b63bf145c8961dbdf10c310efbb3b2676bbc0f8b08505c9e2f795021006b7838609339e8b415a7f9acb1b661828131aef1ecbc7955dfb01f3ca0e","241097efbf8b63bf145c8961dbdf10c310efbb3b2676bbc0f8b08505c9e2f795021006b7838609339e8b415a7f9acb1b661828131aef1ecbc7955dfb01f3ca0e",
+              "240097efbf8b63bf145c8961dbdf10c310efbb3b2676bbc0f8b08505c9e2f795021006b7838609339e8b415a7f9acb1b661828131aef1ecbc7955dfb01f3ca0e","241097efbf8b63bf145c8961dbdf10c310efbb3b2676bbc0f8b08505c9e2f795021006b7838609339e8b415a7f9acb1b661828131aef1ecbc7955dfb01f3ca0e"
```

Comment:
Blocking: this test should actually have the result be `false`, it it just the passing first test with the second byte in the signature set to 0. This is correctly rejected by the C function, but here it's returning `true`. I don't know why this is.